automation_tools/satellite6/hammer.py

```diff
@@ -0,0 +1,98 @@
+import json
```

Comment:
Add a module docstring stating what kind of functions that should be present on this module.